src/sidebar/components/hooks/use-filter-options.js

```diff
@@ -36,16 +41,29 @@ export function useUserFilterOptions() {
     if (focusFilters.user) {
       const username_ =
         username(focusFilters.user.value) || focusFilters.user.value;
-      users[username_] = focusFilters.user.display;
+      const suffix = username_ === currentUsername ? ' (Me)' : '';
+      users[username_] = {
+        display: `${focusFilters.user.display}${suffix}`,
+        value: username_,
+      };
     }
 
-    // Convert to `FilterOption` objects
+    // Convert to an array of `FilterOption` objects
     const userOptions = Object.keys(users).map(user => {
-      return { display: users[user], value: user };
+      return { ...users[user] };
```

Comment:
I refactored this a tad, the structure is now defined above when each item is created. This step is a bit more dumb and just unrolles the Object into an array.  I presume we still need an Object initially to eliminate duplicates.

Reply:
A possible way to do this that would DRY this code out would be to do the "Me" addition here instead of in two prior spots, something like:

```
const userOptions = Object.keys(users).map(user => {
  const display = users[user] === currentUsername ? `${users[user]} (Me)` : users[user];
  return { display, value: user};
});
```

Then you could obliterate the previous two changes...

Reply:
good call, will do.